lib/functional_mixin.js

```diff
@@ -95,6 +108,58 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.swipeLeft = function busterFunctionalMixin_swipeLeft(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
+    this._swipe(target, [-Math.abs(offset), 0], callback);
+
+    return this;
+
+  };
+
+  testCase.swipeRight = function busterFunctionalMixin_swipeRight(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
+    this._swipe(target, [Math.abs(offset), 0], callback);
+
+    return this;
+
+  };
+
+  testCase.swipeUp = function busterFunctionalMixin_swipeUp(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerHeight ?
+                 this.window.innerHeight * this._swipeUpOrDownOffsetFactor :
+                 this._swipeUpOrDownDistance;
```

Comment:
I see ... so I was hoping the swipeUp and down will translate to a scroll, but after our conversation yesterday it clear that swipe up and down will only be triggered on element which can be swiped on. So I agree let me update this with like 2-3 inches. Do you think we need to consider an ipad or tablet swipe.